### ISSUE TITLE
KS.1.1/KP.3.2.3 recombinant (6 seqs, 5 countries)

### ISSUE THREAD

**Issue by aviczhl2** [sars-cov-2-variants/lineage-proposals#1806]
There seems to be a KS.1.1/KP.3.2.3 recombinant, with KS.1.1 Spike. 

KS.1.1--KP.3.2.3

Breakpoint between 23040 and 24820

Additional mutations: C6701T, C19716T, T25510C, C29311T

Orf1a:L2146F, Orf3a:S40P

C6701T, C19716T from a KS.1.1 branch,
T25510C, C29311T from a KP.3.2.3 branch. 

GISAID query: T25510C, C29311T,C19716T,T3565C
No. of seqs: 6(USA 1 Netherlands 1 South Korea 2 Ireland 1 Israel 1)

[usher](https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice114_genome_test_32f78_b09610.json?label=id:node_5642280)
![image](https://github.com/user-attachments/assets/7be1f0be-6742-4718-a961-6c80c69f346b)

**Comment by FedeGueli** [comment]
S:A1087S is reverted so not exactly KS.1.1 spike,  KS.1.1 S1 you likely were meaning

**Comment by aviczhl2** [comment]
> S:A1087S is reverted so not exactly KS.1.1 spike, KS.1.1 S1 you likely were meaning

Yeah and I realize the other donor is not KP.3.2.3 due to not having V1104L.

**Comment by FedeGueli** [comment]
ah good spot!

**Comment by aviczhl2** [comment]
9, Russia

**Comment by FedeGueli** [comment]
+1 NY

**Comment by aviczhl2** [comment]
14, China

**Comment by aviczhl2** [comment]
+17 in the recent South Korean batch, also in Canada, France and Japan

**Comment by aviczhl2** [comment]
17/359=5% prevalence in South Korea

**Comment by FedeGueli** [comment]
56 another batch from South Korea where it is around 5%, please propose it

**Comment by FedeGueli** [comment]
KS.1.1 donor now designated KS.1.1.2 via. https://github.com/cov-lineages/pango-designation/commit/c542eb9c3b5805a189bb9c314dce67744b27ba1e#diff-ead4244ff49c81bc758e9f068f5ef42156bd985b9c86de8fa4d6fcd7f2829090R1277
